tensorflow/BUILD

```diff
@@ -39,7 +39,7 @@ config_setting(
 config_setting(
     name = "android_armeabi",
     values = {
-        "cc_target_os": "android",
+        "crosstool_top": "//external:android/crosstool",
```

Comment:
While this is indeed what we want the BUILD file to look like, this isn't the correct way to make this change.

You haven't done anything wrong; we just have a separate (internal) mechanism to deal with BUILD file differences between internal and open-source builds.

The easiest way forward is for you to revert the change in this file, but keep the change in the other file.  I'll fix up the internal mechanism.  Thanks!